v2.0/drop-sequence.md

```diff
@@ -28,12 +28,12 @@ table td:first-child {
 -----------|------------
```

Comment:
Want to add a comment about the privileges section, but can't do that for some reason, so putting the comment here.

> The user must have the CREATE privilege on the parent database.

Are we sure that's correct? When I just tried to drop a sequence, I got:

```
> jseldess@:26257/> drop sequence test.sequence1;
pq: user jseldess does not have DROP privilege on relation sequence1
```

That implies that the user only nees the `DROP` privilege on the sequence. No mention of the parent database. Maybe @vilterp can clarify?

Reply:
Edited. Will double check with @vilterp

Reply:
Actually not sure about this. Is it different than how tables work? It should be the same.

Reply:
No, `DROP TABLE` works similarly. From the docs:
> The user must have the DROP privilege on the specified table(s). If CASCADE is used, the user must have the privileges required to drop each dependent object as well.

So I think this doc just needs to be corrected, @lhirata.

Reply:
Done.